containers/client-dev/ctia/config/ctia.properties

```
@@ -2,3 +2,4 @@ ctia.store.es.default.host=elasticsearch-client-dev
 ctia.hook.redismq.host=redis-client-dev
 ctia.hook.redis.host=redis-client-dev
 ctia.http.jwt.enabled=false
+ctia.encryption.secret=all_your_bases_are_belong_to_us
```

```suggestion
ctia.encryption.secret=all_your_base_are_belong_to_us
```

https://en.wikipedia.org/wiki/All_your_base_are_belong_to_us